src/GenymotionInstance.js

```diff
@@ -779,6 +779,24 @@ module.exports = class GenymotionInstance {
             }, {
                 widget: this.network,
                 capability: data.message.network,
+            }, {
+                widget: this.network,
+                capability: data.message.mobileThrottling,
+                enable: (widget) => {
+                    widget.enableMobileThrottling();
+                },
+                disable: (widget) => {
+                    widget.disableMobileThrottling();
+                },
+            }, {
+                widget: this.network,
+                capability: data.message.enable5G,
+                enable: (widget) => {
+                    widget.enable5G();
+                },
+                disable: (widget) => {
+                    widget.disable5G();
+                },
```

Comment:
I thought about it, and here is how we could do this in a cleaner way.

1. Add a `setCapabilities()` method in the widget super class. It will store all the capabilities in an array.
2. Add a `refresh()` method in all widgets, that will read the capabilities and update the layout (or disable the widget) accordingly.
3. Loop over the widgets, and call both methods.

This way, at most one refresh of the layouts will ever occur.

Reply:
To do in a future rework, as this is somewhat out of scope.